README.md

```diff
@@ -55,25 +60,30 @@ user@machine:~$ export LABELBOX_API_KEY="<your api key here>"
 user@machine:~$ python3
 
 from labelbox import Client
-client = Client()
+client = Client(api_key="your_key_here", endpoint="http://localhost:8080/_gql")
 ```
 
 ## Contribution
 Please consult `CONTRIB.md`
 
 ## Testing
-1. Update the `Makefile` with your `staging` or `prod` API key. Ensure that docker has been installed on your system. Make sure the key is not from a free tier account.
+1. Update the `Makefile` with your `local`, `staging`, `prod` API key. Ensure that docker has been installed on your system. Make sure the key is not from a free tier account.
 2. To test on `staging`:
 ```
-make test-staging
+make test-staging PATH_TO_TEST=tests/integration/...etc LABELBOX_TEST_API_KEY=specify_here_or_export_me
 ```
 
 3. To test on `prod`:
 ```
-make test-prod
+make test-prod PATH_TO_TEST=tests/integration/...etc LABELBOX_TEST_API_KEY=specify_here_or_export_me
```

Comment:
I feel like everyone should just set a key for each env in their zshrc. Also maybe we can explain that the `PATH_TO_TEST` is optional and by default it runs everything. Also thanks for keeping the README up to date!

Reply:
i get what you're saying, but its just going to need updating the next day, so it just seems a bit easier to update it the command then continuously export it, i suppose?